XCUITests/DataManagementTest.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DataManagementTest.swift
```

Comment:
Please change this header as it is defined in other [tests](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/ActivityStreamTest.swift#L1):